offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/offergeneration/QuotationDetails.groovy

```diff
@@ -200,31 +258,44 @@ class QuotationDetails {
      * @param item The item for which the spacing should be calculated
      * @return The calculated space in the final offer template that the item requires
      */
-    private static int determineItemSpace(OfferItem item) {
+    private static double determineItemSpace(OfferItem item) {
 
-        ArrayList<Integer> calculatedSpaces = []
+        ArrayList<Double> calculatedSpaces = []
 
-        //Determine amount of spacing necessary from highest itemSpace value of all columns
-        calculatedSpaces.add(calculateItemSpace(item.productName, ProductPropertySpacing.PRODUCT_NAME))
-        calculatedSpaces.add(calculateItemSpace(item.productDescription, ProductPropertySpacing.PRODUCT_DESCRIPTION))
-        calculatedSpaces.add(calculateItemSpace(item.quantity as String, ProductPropertySpacing.PRODUCT_AMOUNT))
+        //minimal lineSpace allocated for the item and the space underneath before any linebreaks occur
+        double minimalLineSpace = 0.5
 
-        calculatedSpaces.add(calculateItemSpace(item.unit as String, ProductPropertySpacing.PRODUCT_UNIT))
-        calculatedSpaces.add(calculateItemSpace(item.unitPrice as String, ProductPropertySpacing.PRODUCT_UNIT_PRICE))
-        calculatedSpaces.add(calculateItemSpace(item.getItemTotal() as String, ProductPropertySpacing.PRODUCT_TOTAL))
-        calculatedSpaces.add(calculateItemSpace("Service Provider: "+item.getServiceProvider(), ProductPropertySpacing.PRODUCT_FACILITY))
+        //Determine amount of spacing necessary from highest itemSpace value of all columns
+        calculatedSpaces.add(minimalLineSpace + calculateItemSpace(item.productName, ProductPropertySpacing.PRODUCT_NAME))
+        calculatedSpaces.add(minimalLineSpace + calculateItemSpace(item.quantity as String, ProductPropertySpacing.PRODUCT_AMOUNT))
+        calculatedSpaces.add(minimalLineSpace + calculateItemSpace(item.unit as String, ProductPropertySpacing.PRODUCT_UNIT))
+        calculatedSpaces.add(minimalLineSpace + calculateItemSpace(item.unitPrice as String, ProductPropertySpacing.PRODUCT_UNIT_PRICE))
+        calculatedSpaces.add(minimalLineSpace + calculateItemSpace(item.getItemTotal() as String, ProductPropertySpacing.PRODUCT_TOTAL))
```

Comment:
if you add the minimalLineSpace and it is fixed for all items, why dont you just use it in the calculateItemSpace method?

Reply:
That's a very good point! Nice Catch 👁️.  
This is a remnant when i tried to do a different calculation dependent on the fontsize which as of now is luckily unnecessary